### PR TITLE
[DEV-5136] Minor fix in SQL string

### PR DESCRIPTION
**Pull request by emily-brents**
**Description:**
Error with turning array of ids into sql string.

**Requirements for PR merge:**

1. [X] Unit & integration tests updated
2. [X] API documentation updated
3. [ ] Necessary PR reviewers:
    - [ ] Backend
4. [X] Matview impact assessment completed
5. [X] Frontend impact assessment completed
6. [X] Data validation completed
7. [N/A] Appropriate Operations ticket(s) created
8. [ ] Jira Ticket [DEV-5136](https://federal-spending-transparency.atlassian.net/browse/DEV-5136):
    - [X] Link to this Pull-Request
    - [X] Performance evaluation of affected (API | Script | Download)
    - [X] Before / After data comparison

**Area for explaining above N/A when needed:**
```
```

### CODE REVIEW

File: usaspending_api/idvs/v2/views/amounts.py [hunk 1]
```diff
@@ -59,11 +59,11 @@ def fetch_account_details_idv(award_id, award_id_column) -> dict:
     grandchild_award_ids = []
     child_award_ids.extend([x["award_id"] for x in children])
     grandchild_award_ids.extend([x["award_id"] for x in grandchildren])
-    award_id_sql = "faba.award_id in {award_id}".format(award_id=child_award_ids)
+    award_id_sql = "faba.award_id in {award_id}".format(award_id="(" + str(child_award_ids).strip("[]") + ")")
```
Comment (tony-sappe) [inline usaspending_api/idvs/v2/views/amounts.py:62]: Isn't str(<list>) = "[<list items>]" ?
is the SQL becoming `"faba.award_id in ([child_award_ids])"`

Comment (emily-brents) [reply usaspending_api/idvs/v2/views/amounts.py:62]: no? I don't understand what you mean by `str()="[]"`, we turn the array into a string, strip out the `[` and `]` characters, then add parenthesis.

Comment (tony-sappe) [reply usaspending_api/idvs/v2/views/amounts.py:62]: updated comment to show the list

Comment (tony-sappe) [reply usaspending_api/idvs/v2/views/amounts.py:62]: Tested locally, it looks OK

**Pull request merged**